test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -102,41 +120,85 @@ public void setup() throws Exception {
         this.controllerWrapper.awaitRunning();
         this.controller = controllerWrapper.getController();
 
-        //4. Create Scope
+        // 4. Create Scope
         this.controller.createScope(SCOPE).get();
         ClientConfig clientConfig = ClientConfig.builder().build();
         SocketConnectionFactoryImpl connectionFactory = new SocketConnectionFactoryImpl(clientConfig);
         this.connectionPool = new ConnectionPoolImpl(clientConfig, connectionFactory);
         this.keyValueTableFactory = new KeyValueTableFactoryImpl(SCOPE, this.controller, this.connectionPool);
+
+        // 5. Create segment helper
+        HostMonitorConfig hostMonitorConfig = HostMonitorConfigImpl.builder()
+                .hostMonitorEnabled(false)
+                .hostMonitorMinRebalanceInterval(10)
+                .containerCount(containerCount)
+                .hostContainerMap(HostMonitorConfigImpl.getHostContainerMap(ENDPOINT, servicePort, containerCount))
+                .build();
+        HostControllerStore hostStore = HostStoreFactory.createInMemoryStore(hostMonitorConfig);
+        executor = ExecutorServiceHelpers.newScheduledThreadPool(1, "collector");
```

Comment:
FYI you almost never need to create an Executor in your unit tests. Most unit tests extend from `ThreadPooledTestSuite` which provide you with an executor.